packages/ndla-ui/src/Aside/Aside.tsx

```diff
@@ -15,7 +15,16 @@ const classes = BEMHelper({
   name: 'aside',
 });
 
-const Aside = ({ children, narrowScreen, dangerouslySetInnerHTML, wideScreen }) => {
+interface Props {
+  dangerouslySetInnerHTML?: {
+    __html: string;
+  };
+  children?: React.ReactNode;
+  narrowScreen?: boolean;
+  wideScreen?: boolean;
+}
+
+const Aside = ({ children, narrowScreen = false, dangerouslySetInnerHTML, wideScreen = false }: Props) => {
```

Comment:
Jeg ville kanskje latt `narrowScreen` og `wideScreen` defaulte som `null` slik de var definert i `defaultProps`, siden det er litt vanskelig å teste denne komponenten. Med mindre du har testet det og oppførselen er helt likt

Reply:
Ser ikke komponentet brukt noe sted, så får ikke testet den. Ser den nevnes i `packages/designmanual/stories/simple-components.jsx`, hvor det beskrives at den skal fases ut. 

Gir mening for meg at det er en boolean. Kan bare forestille meg at det har en praktisk forskjell om det eksplisitt sjekkes for om  f.eks. `narrowScreen === null` istedenfor `if (narrowScreen)`. Men ja, litt tricky om man ikke får testet det.

Reply:
Om det er null eller false skal ikke ha noe å si. Se https://www.npmjs.com/package/react-bem-helper